### ISSUE TITLE
Workflow creating .pot file fails because of restrictions on main branch

### ISSUE THREAD

**Issue by linozen** [fsfe/reuse-tool#781]
```plaintext
Run git config --global user.name "GitHub Actions"
[main b3345da] Update reuse.pot
 1 file changed, 43 insertions(+), 36 deletions(-)
remote: error: GH006: Protected branch update failed for refs/heads/main.        
remote: error: Changes must be made through a pull request.        
To https://github.com/fsfe/reuse-tool
 ! [remote rejected] main -> main (protected branch hook declined)
error: failed to push some refs to 'https://github.com/fsfe/reuse-tool'
Error: Process completed with exit code 1.
```

**Comment by carmenbianca** [comment]
I will create a new GitHub account, add it to this org, create a token for it, add the token to this project's secrets, then allow this new account to bypass the protection restrictions.

I will communicate the account credentials with you so that you can add them to the FSFE's password manager (if there is one? If not, to increase the bus factor).

**Comment by mxmehl** [comment]
IIRC we already have a system account called @fsfe-system. The data should already be shared among trusted FSFE System Hackers, @linozen.

**Comment by carmenbianca** [comment]
Should have been fixed by #786, but re-opening this because the action hasn't triggered on the main branch yet.

**Comment by carmenbianca** [comment]
It works!
